### DISCUSSION TITLE
Can I use the free plan of vercel to host my university competition website?

### DISCUSSION THREAD

**Discussion by Syakhisk** [vercel/community#613] [Help] [answered]
Hi,
First of all, thank you for providing free services to be used by developers!

I have a question, can I use the free plan of Vercel to host a next.js app for my university competition?
The website will have:
- Registration form (with fees)
- Sponsor logo at the footer

The requirement said that it is allowed to host a `non-commercial` website and I wonder do the points listed above still comply with the `non-commercial` use?

Thank you

**Comment by samcx** [comment, marked as answer]
Certainly! I don't believe you will need a Pro team with a Sponsorship for this use case.
